contrib/freshmaker-cli

```diff
@@ -356,6 +413,9 @@ parser.add_argument(
 parser.add_argument(
     '--redhat', default=False, action='store_true',
     help="Use the redhat instances.")
+parser.add_argument(
```

Comment:
This would be more intuitive as a [subcommand](https://docs.python.org/3/library/argparse.html#sub-commands) since this flag changes the entire behavior of the command. It doesn't feel like an option to me.

Reply:
Never mind. I misread the PR.